Makefile.inc1

```diff
@@ -1162,10 +1162,16 @@ buildcheri:
 .endif
 
 distributecheri installcheri:
-.for _lib in lib ${LOCAL_LIB_DIRS} cddl/lib gnu/lib secure/lib kerberos5/lib
+.for _lib in lib ${LOCAL_LIB_DIRS} cddl/lib/libctf gnu/lib secure/lib
 	cd ${.CURDIR}/${_lib}; ${LIBCHERIIMAKE} ${.TARGET:S/cheri$//}
 .endfor
 .endif
+.if ${MK_CDDL} != "no"
+	cd ${.CURDIR}/cddl/lib; ${LIBCHERIIMAKE} ${.TARGET:S/cheri$//}
+.endif
+.if ${MK_KERBEROS} != "no"
+	cd ${.CURDIR}/kerberos5/lib; ${LIBCHERIIMAKE} ${.TARGET:S/cheri$//}
```

Comment:
Since this is the same as in the loop I would just do 
```
_installcheri_libs=cddl/lib/libctf gnu/lib secure/lib
.if ${MK_CDDL} != "no"
	_installcheri_libs+=cddl/lib
.endif
...
.for _lib in lib ${LOCAL_LIB_DIRS} ${_installcheri_libs}
````

Reply:
Sounds good.